Packs/UltraMSG/Integrations/UltraMSG/UltraMSG_description.md

```diff
@@ -0,0 +1,4 @@
+### UltraMSG
+ #### Integration Author: Trustnet LTD. [Trustnet Site](https://www.trustnet.co.il)
```

Comment:
Add information about the parameters here.
* How to generate the token?
* What is the `instance` parameter?